src/app/dashboard-widgets/add-codebase-widget/add-codebase-widget.component.html

```diff
@@ -77,13 +91,17 @@ <h2 class="card-pf-title">
         <div class="f8-blank-slate-card" *ngIf="codebases.length === 0" default-level>
           <h3>This Space has no Codebases</h3>
           <div class="f8-blank-slate-main-action">
-            <button id="spacehome-my-codebases-create-button" class="btn btn-primary btn-lg" (click)="addToSpace.emit()">Add Codebase</button>
+            <div *ngIf="userOwnsSpace">
+              <button id="default-level-spacehome-my-codebases-create-button" class="btn btn-primary btn-lg" (click)="addToSpace.emit()">Add Codebase</button>
+            </div>
           </div>
         </div>
         <div class="f8-blank-slate-card" *ngIf="codebases.length === 0" user-level>
           <h3>This Space has no Codebases</h3>
           <div class="f8-blank-slate-main-action">
-            <button class="btn btn-primary btn-lg" (click)="addToSpace.emit()">Add Codebase</button>
+            <div *ngIf="userOwnsSpace">
+              <button id="user-level-spacehome-my-codebases-create-button" class="btn btn-primary btn-lg" (click)="addToSpace.emit()">Add Codebase</button>
+            </div>
           </div>
         </div>
```

Comment:
Lines 91-106

The only change is the button ID. Lets remove the AppLauncher feature here as well.

```
<div class="f8-blank-slate-card" *ngIf="codebases.length === 0" default-level>
          <h3>This Space has no Codebases</h3>
          <div class="f8-blank-slate-main-action">
            <div *ngIf="userOwnsSpace">
              <button id="default-level-spacehome-my-codebases-create-button" class="btn btn-primary btn-lg" (click)="addToSpace.emit()">Add Codebase</button>
            </div>
          </div>
        </div>
        <div class="f8-blank-slate-card" *ngIf="codebases.length === 0" user-level>
          <h3>This Space has no Codebases</h3>
          <div class="f8-blank-slate-main-action">
            <div *ngIf="userOwnsSpace">
              <button id="user-level-spacehome-my-codebases-create-button" class="btn btn-primary btn-lg" (click)="addToSpace.emit()">Add Codebase</button>
            </div>
          </div>
</div>
```